timelock-server/src/main/java/com/palantir/atlasdb/timelock/paxos/PaxosResource.java

```diff
@@ -49,16 +51,32 @@ public static PaxosResource create() {
     }
 
     public static PaxosResource create(String logDirectory) {
-        return new PaxosResource(logDirectory, Maps.newHashMap(), Maps.newHashMap());
+        return new PaxosResource(logDirectory, Maps.newConcurrentMap(), Maps.newConcurrentMap());
```

Comment:
Hmm, do the maps actually need to be thread-safe? Might be wrong, but I thought the Paxos resource was built on startup and the client -> learner/acceptor maps were constructed at that time and never subsequently changed.

(Not a blocker - I'm just curious regarding this change)